calico_node/tests/st/utils/docker_host.py

```diff
@@ -319,12 +321,16 @@ def start_calico_node(self, options="", with_ipv4pool_cidr_env_var=True):
             # Break the line at the first occurrence of " -e ".
             prefix, _, suffix = line.rstrip().partition(" -e ")
 
+            felix_logsetting = ""
+            if FELIX_LOGLEVEL != "":
+                felix_logsetting = " -e FELIX_LOGSEVERITYSCREEN=" + FELIX_LOGLEVEL
+
             # Construct the calicoctl command that we want, including the
             # CALICO_IPV4POOL_CIDR setting.
             modified_cmd = (
                 prefix +
                 (" -e CALICO_IPV4POOL_CIDR=%s " % DEFAULT_IPV4_POOL_CIDR) +
-                " -e DISABLE_NODE_IP_CHECK=true -e FELIX_IPINIPENABLED=true " +
+                " -e DISABLE_NODE_IP_CHECK=true" + felix_logsetting + " -e FELIX_IPINIPENABLED=true " +
```

Comment:
I think you should add this on its own line because I expect we will remove the DISABLE_NODE_IP_CHECK and FELIX_IPIPENABLED flags at some point (soon) and I'd rather the two changes not be mixed.